qcengine/programs/psi4.py

```diff
@@ -7,7 +7,8 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, ClassVar, Dict
 
-from qcelemental.models.v2 import AtomicResult, BasisSet
+from qcelemental.models.v1 import AtomicResult as AtomicResult
+from qcelemental.models.v2 import BasisSet
```

Comment:
Is `BasisSet` being used?

Reply:
Yes, albeit briefly https://github.com/MolSSI/QCEngine/pull/458/files#diff-d66842b418a1c03f41438563803c987c9defd1811ea0af131e285ba540c8c0d6R154